draft-amsuess-core-shopinc.md

```diff
@@ -138,7 +146,7 @@ or cause a 4.02 Bad Option error).
 
 Examples of rules are:
 
-* Options after the first are treated exactly like Uri-Path options.
+* Options after the first are treated exactly like Uri-Path options. (Hereby called the initial+path rule)
```

Comment:
With the new text, the whole "examples of rules" can go out; let's have one or two places only to list them (currently it's two, the text in the option description and the choices for IANA, but I'd prefer to unify them.)